src/atn/ATN.ts

```diff
@@ -78,21 +122,38 @@ export class ATN {
 	 */
 	lexerActions: LexerAction[];
 
+	/**
+	** In addition to names, Lexer modes also have numeric identifiers for faster lookup.
+	*/
 	@NotNull
 	modeToStartState: TokensStartState[] = [];
 
 	private contextCache: Array2DHashMap<PredictionContext, PredictionContext> =
 		new Array2DHashMap<PredictionContext, PredictionContext>(ObjectEqualityComparator.INSTANCE);
 
+	/**
+	 * Each decision point in the grammar may be represented by a cached DFA,
+	 * but the cached DFAs are mutable.
+	 */
 	@NotNull
 	decisionToDFA: DFA[] = [];
 	@NotNull
 	modeToDFA: DFA[] = [];
 
+<<<<<<< 349fcddba4bcad149b0192276637e87e91111f44
```

Comment:
Oh yes, hmm, I wonder why wasn't that obvious in when I looked at the appveyor output.